assets/javascripts/modules/analytics/omniture.js

```diff
@@ -51,6 +53,18 @@ define([
         s.eVar19 = 'D=c19';
         s.prop42 = 'Subscriber';
         s.prop47 = 'UK';
+
```

Comment:
Comment to explain that you're just extracting any attribute containing data-tracking-? Took me a bit of squinting to figure it out